examples/notebooks/osft_dataset_scaling_guide.ipynb

```
@@ -0,0 +1,758 @@
+      "outputs": [
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "/mnt/7TB-a/osilkin/non-critical-changes/training_hub/.venv/lib/python3.12/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
+            "  from .autonotebook import tqdm as notebook_tqdm\n"
+          ]
+        }
+      ],
+      "source": [
+        "# Import training_hub for OSFT training\n",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)